rero_ils/query.py

```diff
@@ -37,23 +37,26 @@ def document_search_factory(self, search, query_parser=None):
     agg_size = current_app.config.get('RERO_ILS_AGGREGATION_SIZE').get(
         'documents', current_app.config.get(
             'RERO_ILS_DEFAULT_AGGREGATION_SIZE'))
-    view = request.args.get(
-        'view', current_app.config.get('RERO_ILS_SEARCH_GLOBAL_VIEW_CODE'))
-    if view == current_app.config.get('RERO_ILS_SEARCH_GLOBAL_VIEW_CODE'):
+    view = request.args.get('view')
+    if view is None:
```

Comment:
Do we have tree cases here?
1. view is None
2. view == current_app.config(...
3. else

Reply:
Yes, are you proposing some improvement?

Reply:
There is no switch in python only:
if .....
elif .....
elif .....